src/templates/user/settings/metatrader.html.tt

```diff
@@ -69,7 +69,7 @@
                 <p id="msg-cannot-create-real" class="notice-msg center-text invisible"></p>
                 <div id="authenticate" class="center-text invisible">
                     <p>[% l('To create a Real account for MetaTrader, you need to authenticate your Binary account:') %]</p>
-                    <a class="button pjaxload" href="[% request.url_for('/user/authenticatews') %]">
+                    <a class="button pjaxload" href="https://www.binary.com/en/user/authenticatews.html" target="_blank">
```

Comment:
Sure, I will change it.

On 31 Aug 2016 9:46 a.m., "Ashkan" <notifications@github.com> wrote:

> In src/templates/user/settings/metatrader.html.tt
> <https://github.com/binary-com/binary-mt/pull/10#discussion_r76911660>:
>
> > @@ -69,7 +69,7 @@
> >                  <p id="msg-cannot-create-real" class="notice-msg center-text invisible"></p>
> >                  <div id="authenticate" class="center-text invisible">
> >                      <p>[% l('To create a Real account for MetaTrader, you need to authenticate your Binary account:') %]</p>
> > -                    <a class="button pjaxload" href="[% request.url_for('/user/authenticatews') %]">
> > +                    <a class="button pjaxload" href="https://www.binary.com/en/user/authenticatews.html" target="_blank">
>
> Can you NOT hardcode the language?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/binary-com/binary-mt/pull/10/files/d195dd2a6773f66f69ad388b047fabbbe1ee95ad#r76911660>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIiaeBmsWqG-pLLxIXm9-wR7sknnf4M_ks5qlNzxgaJpZM4JwV7R>
> .
>